specutils/analysis/flux.py

```diff
@@ -193,7 +193,17 @@ def _compute_equivalent_width(spectrum, continuum=1, regions=None,
     dx = np.abs(np.diff(spectrum.spectral_axis.bin_edges))
     ew = np.sum((1 - line_flux / cont_flux) * dx)
 
-    return ew.to(spectrum.spectral_axis.unit)
+    # NOTE: uncertainty gets lost during unit translation, so we have to convert first
+    ew = ew.to(spectrum.spectral_axis.unit)
+    if line_flux.uncertainty is not None:
+        # continuum has no uncertainty, here we assume no uncertainty on the bin edges (dx) as well
```

Comment:
We should probably *eventually* provide a way to give uncertainty through the continuum. Problem is, I'm not sure what the best way is. I would be OK with any of:
1. Allow the `continuum` keyword to be an `astropy.uncertainty`, although that doesn't really play smoothly yet with nddata uncertainties
2. Add a `continuum_uncertainty` keyword (or `continuum` as a 2-tuple?) that uses the NDData uncertainty objects to encode the continuum uncertainty
3. punt the problem for the future and record it as an issue.

Regardless of which is chosen, though, the docstring should be amended to explicitly say that continuum uncertainty is assumed to be 0.